libcontainer/system/proc_test.go

```diff
@@ -43,6 +43,36 @@ func TestParseStartTime(t *testing.T) {
 	}
 }
 
+func TestParseStatBadInput(t *testing.T) {
+	cases := []struct {
+		desc, input string
+	}{
+		{"no (",
+			"123 ) S 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0"},
+		{"no )",
+			"123 ( S 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0"},
+		{"negative pid",
+			"-1 (cmd) S 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0"},
+		{"empty line",
+			""},
+		{"short line",
+			"123 (cmd) S 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0"},
+		{"short line (no space after stime)",
+			"123 (cmd) S 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 42"},
+		{"bad stime",
+			"123 (cmd) S 2 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 -1 "},
+		{"bad stime 2", // would be valid if not -1
+			"123 (cmd) S                   -1 "},
+	}
+	for _, c := range cases {
+		st, err := parseStat(c.input)
+		t.Logf("case %q: error: %v, stat: %v", c.desc, err, st)
```

Comment:
This t.Logf needs to be removed. We have enough output in our CI logs already :)

Reply:
fixed